docs/self-hosted/oel/kratos/upgrade.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-id: upgrade-kratos
-title: Upgrade Ory Kratos OEL
+id: upgrade
+title: Upgrade Ory Kratos OEL to the newer version
```

Comment:
needs a shorter sidebar title

Reply:
wdym with 2 items? 
I only see this doc: https://docs-2m9ztxavw-ory.vercel.app/docs/self-hosted/oel/kratos/upgrade

I think the title change is fine, it just looks a bit long in the sidebar? 

<img width="320" alt="Screenshot 2025-05-30 at 16 25 36" src="https://github.com/user-attachments/assets/f94db306-0fc1-421c-9b26-78691a9f9c43" />

IMO "Upgrade Ory Kratos OEL" is fine here as sidebar title or what am I missing?